packages/server/src/automations/tests/scenarios/scenarios.spec.ts

```diff
@@ -444,43 +444,48 @@ describe("Automation Scenarios", () => {
       },
     ]
 
-    testCases.forEach(({ condition, value, rowValue, expectPass }) => {
-      it(`should ${
-        expectPass ? "pass" : "fail"
-      } the filter when condition is "${condition}" and value is ${value}`, async () => {
-        const builder = createAutomationBuilder({
-          name: `Test ${condition}`,
-        })
-
-        const results = await builder
-          .appAction({ fields: {} })
-          .createRow({
-            row: {
-              name: `${condition} Test`,
-              value: rowValue,
-              tableId: table._id,
-            },
-          })
-          .queryRows({
-            tableId: table._id!,
-          })
-          .filter({
-            field: "{{ steps.2.rows.0.value }}",
-            condition,
-            value,
+    it.each(testCases)(
+      "should pass the filter when condition is %s",
+      async ({ condition, value, rowValue, expectPass }) => {
+        it(`should ${
+          expectPass ? "pass" : "fail"
+        } the filter when condition is "${condition}" and value is ${value}`, async () => {
```

Comment:
You shouldn't need this anymore, I suspect this doesn't actually work as-is.

Reply:
I was... not concentrating at all when I made the last commit. Just left the existing code under it lol, should be sorted now!